src/hiero_sdk_python/tokens/token_airdrop_claim.py

```
@@ -197,14 +202,8 @@ def __str__(self) -> str:
-            for aid in self._pending_airdrop_ids
-        ]
+        lines = [f"   → {aid}" for aid in self._pending_airdrop_ids]
 
         ids_block = "\n".join(lines)
-        summary = (
-            f"Pending Airdrops to claim:\n"
-            f"{ids_block}\n"
-        )
-        return summary
\ No newline at end of file
+        summary = f"Pending Airdrops to claim:\n" f"{ids_block}\n"
```

Ruff (0.14.14)

[warning] Implicitly concatenated string literals on one line

Combine string literals

(ISC001)